abletonosc/clip.py

```diff
@@ -70,27 +70,49 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             "remove_notes_by_id"
         ]
         properties_r = [
+            ##"end_time" ## doesn't seem to do anything, can't test
             "file_path",
             "gain_display_string",
+            ##"groove", ## if other than None, says "Error handling OSC message: Infered arg_value type is not supported"
+            "has_groove", ## False/True
```

Comment:
Don't need comments for return types here, please remove for consistency with the rest of the code